projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-list-users-endpoint.php

```diff
@@ -181,14 +181,16 @@ public function callback( $path = '', $blog_id = 0 ) {
 		) : array();
 		$viewers = array_map( array( $this, 'get_author' ), $viewers );
 
-		// we restrict search field to name when include_viewers is true.
+		// When include_viewers is true, search by username or email.
 		if ( $include_viewers && ! empty( $args['search'] ) ) {
 			$viewers = array_filter(
 				$viewers,
 				function ( $viewer ) use ( $args ) {
+					// Convert to WP_User so expected fields are available.
+					$wp_viewer = new WP_User( $viewer->ID );
 					// remove special database search characters from search term
 					$search_term = str_replace( '*', '', $args['search'] );
-					return strpos( $viewer->name, $search_term ) !== false;
+					return ( strpos( $wp_viewer->user_login, $search_term ) !== false || strpos( $wp_viewer->user_email, $search_term ) !== false );
```

Comment:
Would it be an option to rely on `str_contains` here?

Reply:
In terms of what's returned it shouldn't make a difference. Would using `str_contains` have benefits relative to `strpos`?

Reply:
> Would using `str_contains` have benefits relative to `strpos`?

I find it more readable, and we've tried to follow Core's lead on that:
https://core.trac.wordpress.org/ticket/58206

Reply:
That makes sense! Updated.